README.md

```diff
@@ -13,3 +13,97 @@ Please see the [contributing guide](https://github.com/exercism/docs/tree/master
+
+- Follow the coding standards for Rust.  [rustfmt](https://github.com/nrc/rustfmt) may help with this
+and can be installed with `cargo install rustfmt`.
+
+- Watch out for trailing spaces, extra blank lines, and spaces in blank lines.
+
+- All the tests for Rust exercises can be run from the top level of the repo with `_test\check-exercises.sh` Please run this command before submitting your PR.  See [Windows Specific instructions](_test/WINDOWS_README.md) for running this.
+
+## Contributing a New Exercise ##
+
+- All Exercism exercises must be defined in [problem-specifications](https://github.com/exercism/problem-specifications/tree/master/exercises) before they are implemented for a specific track. Please submit a PR there if your exercise is new to Exercism.
+
+- Please make sure the new exercise conforms to specifications in the [exercism/problem-specifications](https://github.com/exercism/problem-specifications) repo.  (Note if slight changes are needed for Rust
+specific issues, see `.meta\hints.md below.)
+
+- Each exercise must stand on its own. Do not reference files outside the exercise directory. They will not be included when the user fetches the exercise.
+
+- Exercises should use only the Rust core libraries.  If you must use an external crate, see note below about `Cargo-example.toml`.  `Cargo.toml` should not have external dependencies as we don't want to make the user assume which crates are required.
+
+- Each exercise should have:
+---
+    exercises/exercise-name/
+                            tests/exercise-name.rs  <- a test suite
+                            src/lib.rs              <- an empty file or with exercise stubs
+                            example.rs              <- example solution that satisfies tests
+                            Cargo.toml              <- with version equal to exercise defintion
+                            README.md               <- Instructions for the exercise (see notes below)
+---
+
+- An exercise may contain `.meta/hints.md`.  This is optional and will appear after the normal exercise
+  instructions if present.  Rust is different in many ways from other languages.  This is a place where 
+  the differences required for Rust are explained.  If it a large change, you may want to call this out
+  as a comment at the top of `src/lib.rs`, so the user recognises to read this section before starting.
+
+- If an `example.rs` uses external crates, include `Cargo-example.toml` so that `_tests\check-exercises.sh` can compile with these when testing.
+
+- We are currently committing Cargo.lock.  This may change.
+
+- `README.md` may be regenerated from exercism data.  The top section above `## Rust Installation` should contain `description.md` from the exercise directory in the 
+[problem-specifications github.](https://github.com/exercism/problem-specifications/tree/master/exercises)  The `## Source` section comes from the `metadata.yml` in the same directory.
```

Comment:
I would say "problem-specifications repository" instead of "problem-specifications github" (since it may not always be clear what github means when used in that context)